src/components/TheNav.vue

```diff
@@ -128,11 +131,23 @@ export default {
     ...mapActions(useUserStore, ['signOut']),
     toggleNav(cb) {
       this.isNavShown = !this.isNavShown
+      
+      if (this.isNavShown) {
+        this.isLocked = true
+      } else {
+        this.isLocked = false
+      }
```

Comment:
```suggestion
        this.isLocked = this.isNavShown
```

Reply:
🙈

I could think of it yesterday but today no...

**Corrected**